server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -1440,6 +1452,11 @@ public static Settings addHumanReadableSettings(Settings settings) {
             ZonedDateTime creationDateTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(creationDate), ZoneOffset.UTC);
             builder.put(SETTING_CREATION_DATE_STRING, creationDateTime.toString());
         }
+        Long lifecycleOriginationDate = settings.getAsLong(SETTING_LIFECYCLE_ORIGINATION_DATE, null);
```

Comment:
This code is also probably not needed since we don't want x-pack settings in the `server` module